schemas/codify.xsd

```diff
@@ -1,50 +1,215 @@
+                    <xs:attribute name="technical" type="xs:string" use="optional"/>
+                    <xs:attribute name="warning" type="falseOnlyType" use="optional"/>
+                </xs:extension>
+            </xs:simpleContent>
+        </xs:complexType>
+    </xs:element>
+    <xs:element name="insert">
+        <xs:complexType>
+            <xs:complexContent>
+                <xs:extension base="insertType">
+                    <xs:attributeGroup ref="lib:targetAttributes"/>
+                    <xs:attributeGroup ref="historyAttributes"/>
+                    <xs:attribute name="after" type="xs:string" use="optional"/>
+                    <xs:attribute name="before" type="xs:string" use="optional"/>
+                    <xs:attribute name="num-value" type="xs:string" use="optional"/>
+                    <xs:attribute name="technical" type="xs:string" use="optional"/>
+                    <xs:attribute name="warning" type="falseOnlyType" use="optional"/>
+                </xs:extension>
+            </xs:complexContent>
+        </xs:complexType>
+    </xs:element>
+    <xs:element name="insert-code-container" type="anyType"/>
+    <xs:element name="not-funded-anno">
+        <xs:complexType>
+            <xs:simpleContent>
+                <xs:extension base="lib:emptyType">
+                    <xs:attributeGroup ref="lib:targetAttributes"/>
+                    <xs:attributeGroup ref="historyAttributes"/>
+                    <xs:attribute name="action" type="xs:string" use="optional"/>
+                    <xs:attribute name="applicability-path" type="xs:string" use="optional"/>
+                    <xs:attribute name="warning" type="falseOnlyType" use="optional"/>
+                </xs:extension>
+            </xs:simpleContent>
+        </xs:complexType>
+    </xs:element>
+    <xs:element name="organic">
+        <xs:complexType>
+            <xs:complexContent>
+                <xs:extension base="insertType">
+                    <xs:attributeGroup ref="lib:targetAttributes"/>
+                    <xs:attributeGroup ref="historyAttributes"/>
+                    <xs:attribute name="after" type="xs:string" use="optional"/>
+                    <xs:attribute name="before" type="xs:string" use="optional"/>
+                    <xs:attribute name="heading-value" type="xs:string" use="optional"/>
+                    <xs:attribute name="num-value" type="xs:string" use="optional"/>
+                    <xs:attribute name="prefix-value" type="xs:string" use="optional"/>
+                    <xs:attribute name="tag" type="xs:string" use="optional"/>
+                    <xs:attribute name="technical" type="xs:string" use="optional"/>
+                    <xs:attribute name="warning" type="falseOnlyType" use="optional"/>
+                </xs:extension>
+            </xs:complexContent>
+        </xs:complexType>
+    </xs:element>
+    <xs:element name="redesignate" type="anyType"/>
```

Comment:
This does not seem to be used anywhere (`<codify:redesignate-para>` is used. however). Should this be removed?